src/components/organisms/Settings/ProjectList/hooks.ts

```diff
@@ -23,12 +25,15 @@ const toPublishmentStatus = (s: PublishmentStatus) =>
 export default () => {
   const [currentTeam, setLocalState] = useLocalState(s => s.currentTeam);
   const navigate = useNavigate();
+  const intl = useIntl();
 
   const [modalShown, setModalShown] = useState(false);
   const openModal = useCallback(() => setModalShown(true), []);
 
   const { data, loading, refetch } = useMeQuery();
-  const [createNewProject] = useCreateProjectMutation();
+  const [createNewProject] = useCreateProjectMutation({
+    refetchQueries: ["Project"],
```

Comment:
If we are doing a refetch() in the createProject function for MEQuery do we need the refetchQuery attached here? 
Pretty sure we just need one (and the refetch() makes sense since we are fetching AFTER the project is created AND added to the scene).
Please let me know if I am mistaking the logic!

Reply:
Like yesterday I pairprogram with Basel , we found seems the MEQuery didn't update the projects ... that's why here to refetch the projects. I hope we can have a discuss @KaWaite @Basel-Issmail for this problem